### DISCUSSION TITLE
hubserving部署structure_system时，为什么提示ValueError: not find model.pdmodel or inference.pdmodel in

### DISCUSSION THREAD

**Discussion by asenasen123** [PaddlePaddle/PaddleOCR#14226] [Q&A]
模型应该都在，其他模块都可以正常部署，但部署structure_system时，出现上述问题

**Comment by GreatV** [comment]
hubserving 已经不维护，建议不要使用。

**Reply by asenasen123** [reply]
> hubserving 已经不维护，建议不要使用。
那如果要部署ppstruct模型，该选用什么方式部署？pdserving好像也不行？

**Reply by GreatV** [reply]
可以试试paddlex团队的服务化部署 https://paddlepaddle.github.io/PaddleX/latest/pipeline_deploy/service_deploy.html

**Reply by asenasen123** [reply]
> 可以试试paddlex团队的服务化部署 https://paddlepaddle.github.io/PaddleX/latest/pipeline_deploy/service_deploy.html

请问下，paddlex可以实现版面分析、检测、识别、表格提取pipeline吗？

**Reply by GreatV** [reply]
可以参考通用OCR相关产线体验 https://paddlepaddle.github.io/PaddleX/latest/pipeline_usage/tutorials/ocr_pipelines/OCR.html#ocr
